docs/semgrep-code/remove-duplicates.md

```diff
@@ -5,26 +5,116 @@ title: Remove duplicate findings
 hide_title: true
 description: Learn how to remove duplicate findings and prevent them from displayed in Semgrep AppSec Platform.
 tags:
-  - Semgrep Code
-  - Semgrep AppSec Platform
+ - Semgrep Code
+ - Semgrep AppSec Platform
 ---
 
 # Remove duplicate findings
 
 Semgrep scans are performed on both mainline (trunk) and non-mainline branches. The scope of the scan can differ depending on if Semgrep is called on a mainline or non-mainline branch.
 
 <dl>
-    <dt>Full scan</dt>
-    <dd>Scans the repository in its entirety. It is recommended to perform full scans on mainline branches, such as <code>master</code> or <code>main</code>. This scan is performed on a scheduled basis.</dd>
-    <dt>Diff-aware scan</dt>
-    <dd>Diff-aware scans are performed on non-mainline branches, such as in pull requests and merge requests. Diff-aware scans traverse the repository's files based on the commit where the branch diverged from the mainline branch (or diverged from the last commit that was fully scanned)</dd>
+ <dt>Full scan</dt>
+ <dd>Scans the repository in its entirety. It is recommended to perform full scans on mainline branches, such as <code>master</code> or <code>main</code>. This scan is performed on a scheduled basis.</dd>
+ <dt>Diff-aware scan</dt>
+ <dd>Diff-aware scans are performed on non-mainline branches, such as in pull requests and merge requests. Diff-aware scans traverse the repository's files based on the commit where the branch diverged from the mainline branch (or diverged from the last commit that was fully scanned)</dd>
 </dl>
 
-## Remove duplicate findings using Semgrep AppSec Platform
+## How Semgrep distinguishes between new and duplicate findings
 
-Regardless of the scope of a scan, Semgrep correlates findings across branches based on their unique fingerprint, automatically deduplicating findings and making it simpler to triage.
+Semgrep generates a finding whenever it scans a project, and one of its rules matches a piece of code in the project. Since Semgrep scans a project multiple times, it needs a way to track the same finding in a file over time. Semgrep does this using two types of fingerprints: `syntactic_id` and `match_based_id`.
```

Comment:
```suggestion
Semgrep generates a finding whenever it scans a project (repository), and one of its rules matches a piece of code in the project. Since Semgrep scans a project multiple times, it needs a way to track the same finding in a file over time. Semgrep does this using two types of fingerprints: `syntactic_id` and `match_based_id`.
```
Or, I think we refer to the scan target as a repository in the above section, so it can also be "repository or project" - you decide!